layouts/partials/section-index.html

```diff
@@ -4,7 +4,8 @@
     {{ $pages = (where $pages "Type" "!=" "search") }}
     {{ $pages = (where $pages ".Params.hide_summary" "!=" true) -}}
     {{ $pages = (where $pages ".Parent" "!=" nil) -}}
-    {{ if and .Parent .Parent.File -}}
+    {{ $pages = (where $pages ".Parent.File" "!=" nil) -}}
+    {{ if and $parent $parent.File -}}
```

Comment:
After looking at this more closely, it seems to make sense indeed.

Btw, the `$parent` variable is misnamed, because`$parent` is set to `.Page`.

Can we inline the variable instead?

Btw, IMHO `.Page` cannot be falsy, but `.Page.File` could be falsy. So the if condition can be simplified.

Reply:
Will track separately:

- #1946